package/gluon-config-mode-mesh-vpn/luasrc/lib/gluon/config-mode/reboot/0100-mesh-vpn.lua

```diff
@@ -1,33 +1,33 @@
 local uci = luci.model.uci.cursor()
 local meshvpn_enabled = uci:get("fastd", "mesh_vpn", "enabled", "0")
 
-if meshvpn_enabled ~= "1" then
-  return nil
-else
-  local i18n = require "luci.i18n"
-  local util = require "luci.util"
+local i18n = require "luci.i18n"
+local util = require "luci.util"
 
-  local gluon_luci = require 'gluon.luci'
-  local site = require 'gluon.site_config'
-  local sysconfig = require 'gluon.sysconfig'
+local gluon_luci = require 'gluon.luci'
+local site = require 'gluon.site_config'
+local sysconfig = require 'gluon.sysconfig'
 
-  local pretty_hostname = require 'pretty_hostname'
+local pretty_hostname = require 'pretty_hostname'
 
-  local pubkey = util.trim(util.exec("/etc/init.d/fastd show_key " .. "mesh_vpn"))
-  local hostname = pretty_hostname.get(uci)
-  local contact = uci:get_first("gluon-node-info", "owner", "contact")
+local pubkey = util.trim(util.exec("/etc/init.d/fastd show_key " .. "mesh_vpn"))
```

Comment:
Keep the `pubkey` variable unset here and only call the `show_key` command when the mesh VPN is enabled. Otherwise, this will unnecessarily generate the key.